cache/cloudfront_prod.tf

```diff
@@ -245,6 +245,6 @@ resource "aws_cloudfront_distribution" "wellcomecollection_org" {
     }
   }
 
-  retain_on_delete = true
+  retain_on_delete = false
```

Comment:
These are not actual infra changes - the CF distro was out of sync - this represents the real state of CF.

Reply:
See custom error responses too ^